internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -4819,59 +4819,73 @@ func TestHandleGetTreasuryBalance(t *testing.T) {
 			Balance: balance,
 			Updates: updates,
 		},
-	}, {
-		name:    "handleGetTreasuryBalance: ok with block hash",
-		handler: handleGetTreasuryBalance,
-		cmd: &types.GetTreasuryBalanceCmd{
-			Hash: &blkHashString,
-		},
-		result: types.GetTreasuryBalanceResult{
-			Hash:    blkHashString,
-			Height:  blkHeight,
-			Balance: balance,
-		},
-	}, {
-		name:    "handleGetTreasuryBalance: invalid hex",
-		handler: handleGetTreasuryBalance,
-		cmd: &types.GetTreasuryBalanceCmd{
-			Hash: dcrjson.String("invalid hex"),
-		},
-		wantErr: true,
-		errCode: dcrjson.ErrRPCDecodeHexString,
-	}, {
-		name:    "handleGetTreasuryBalance: block not found",
-		handler: handleGetTreasuryBalance,
-		cmd:     &types.GetTreasuryBalanceCmd{},
-		mockChain: func() *testRPCChain {
-			chain := defaultMockRPCChain()
-			chain.treasuryBalanceErr = blockchain.ErrUnknownBlock
-			return chain
-		}(),
-		wantErr: true,
-		errCode: dcrjson.ErrRPCBlockNotFound,
-	}, {
-		name:    "handleGetTreasuryBalance: treasury inactive for block",
-		handler: handleGetTreasuryBalance,
-		cmd:     &types.GetTreasuryBalanceCmd{},
-		mockChain: func() *testRPCChain {
-			chain := defaultMockRPCChain()
-			chain.treasuryBalanceErr = blockchain.ErrNoTreasuryBalance
-			return chain
-		}(),
-		wantErr: true,
-		errCode: dcrjson.ErrRPCNoTreasury,
-	}, {
-		name:    "handleGetTreasuryBalance: failed to obtain treasury balance",
-		handler: handleGetTreasuryBalance,
-		cmd:     &types.GetTreasuryBalanceCmd{},
-		mockChain: func() *testRPCChain {
-			chain := defaultMockRPCChain()
-			chain.treasuryBalanceErr = errors.New("failed to obtain treasury balance")
-			return chain
-		}(),
-		wantErr: true,
-		errCode: dcrjson.ErrRPCInternal.Code,
-	}})
+	},
+		{
```

Comment:
The formatting is messed up here.  Should be up a line as it was.

Reply:
Just saw that too.  Fix pushed.